task/sast-snyk-check-oci-ta/0.3/sast-snyk-check-oci-ta.yaml

```diff
@@ -196,6 +204,19 @@ spec:
           echo "Results:"
           (set -x && csgrep --mode=evtstat sast_snyk_check_out_all_findings.json)
 
+          if [[ "${KFP_GIT_URL}" == "SITE_DEFAULT" ]]; then
+            case "${BUILD_PLR_LOG_URL}" in
+            *console.redhat.com* | *console.dev.redhat.com*)
```

Comment:
Sorry for the confusion in #1849.  I thought that `console.redhat.com` was the internal Konflux cluster.  Nevertheless, I would invert the logic to make it more community-friendly.

Could we please match the internal Konflux instances and fallback to empty string for everything else?

If a new Konflux cluster appears in the future, it is safer to assume that it does not have access to RH internal network.

Reply:
Updated.